modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6767,6 +6767,7 @@ enable-template-creation-description=If checked, enable the creation of a dynami
 enable-tensorflow-image-auto-tagging-on-this-instance=Enable TensorFlow Image Auto Tagging on This Instance
 enable-terms-of-use=Enable Terms of Use
 enable-this-for-resources-that-are-hosted-on-a-different-domain=Enable this for resources that are hosted on a different domain.
+enable-to-retain-active-facet-selections-across-searches=Enable to retain active facet selections across searches. When disabled, facet selections will be cleared on new searches.
```

Comment:
How about: "Enable to retain facet selections across searches. When disabled, facets will be reset when a new keyword is searched."